fineract-provider/src/main/java/org/apache/fineract/portfolio/account/api/AccountTransfersApiResourceSwagger.java

```diff
@@ -370,7 +370,7 @@ private GetAccountTransfersGroups() {}
             }
 
             @Schema(example = "1")
-            public Integer id;
```

Comment:
This looks wrong to me...It should be Long based on the GroupGeneralData line:42

Reply:
Done.